src/Integration/Helpers/IDEWindowService.cs

```diff
@@ -0,0 +1,34 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.ComponentModel.Composition;
+using SonarLint.VisualStudio.Core;
+
+namespace SonarLint.VisualStudio.Integration.Helpers
+{
+    [Export(typeof(IIDEWindowService))]
+    internal class IDEWindowService : IIDEWindowService
+    {
+        public void BringToFront()
+        {
+            // TODO
```

Comment:
There is an existing VS "BringToFront" command, but it doesn't seem to work reliably i.e. the IDE window isn't always brought to the top. Mind you, directly calling the WinApi doesn't seem always work either...

Reply:
Hm, I wonder if it's what I experienced, that I told you about before: that sometimes I click on "Open in explorer" and the window is not brought to front 🤔 I think it could be some windows limitation.

Reply:
Also, I added this empty implementation for now otherwise it would break the dogfood experience.

Reply:
No, it doesn't flash - there's no indication that anything has happened :-(